spec/trimmer_decorator_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../trimmer_decorator'
+require_relative '../person'
+require_relative '../base_decorator'
+require 'rspec'
+
+describe TrimmerDecorator do
```

Comment:
- [ Required ] I think it would be better if you add tests for the `correct_name` method in the `TrimmerDecorator` class, according to the project requirements you are expected to test all methods in the classes you created including the `TrimmerDecorator`. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-06-09 00-19-48](https://github.com/LoviMadaan/oop-school-library/assets/16392046/e50b3362-7540-4d9b-a450-83a1b4c934ba)